docs/source/manual/servlets.rst

```diff
@@ -14,12 +14,52 @@ HealthCheckServlet
 ``HealthCheckServlet`` responds to ``GET`` requests by running all the [health checks](#health-checks)
 and returning ``501 Not Implemented`` if no health checks are registered, ``200 OK`` if all pass, or
 ``500 Internal Service Error`` if one or more fail. The results are returned as a human-readable
-``text/plain`` entity.
+``application/json`` entity.
```

Comment:
Maybe we could just write "as a human-readable JSON entity"?
```suggestion
JSON entity.
```

Reply:
Updated - thanks!